pkg/vmprovider/providers/vsphere2/virtualmachine/backup.go

```diff
@@ -125,14 +125,16 @@ func BackupVirtualMachine(opts BackupVirtualMachineOptions) error {
 	}
 
 	if len(ecToUpdate) != 0 {
-		opts.VMCtx.Logger.Info("Updating VM ExtraConfig with latest backup data",
-			"ExtraConfigToUpdate", ecToUpdate)
-		if _, err := opts.VcVM.Reconfigure(opts.VMCtx, types.VirtualMachineConfigSpec{
+		opts.VMCtx.Logger.Info("Updating VM ExtraConfig with latest backup data")
```

Comment:
I would still print the EC that we are trying to update.  Could be handy in failure cases.

Reply:
Updated to log it before the reconfigure call.